### DISCUSSION TITLE
InvalidKey (google maps api ) in deployment

### DISCUSSION THREAD

**Discussion by zorzpapaduby** [vercel/community#2784] [Help] [answered]
### Summary

After deploying in Vercel my Google Maps stopped working. I am getting InvalidKey error.

### Steps to Reproduce

Hello! 
I have used <LoadScritp> from @react-google-maps/api. Everything was working on my localhost, but now when I deployed it on Vercel I get error that my key is Invalid. I have already put the key in Environment Variable.

Note: Interesting in order to make maps work on localhost I had to add to "" at the beginning and end of api key in my env file.

for example NEXT_PUBLIC_GOOGLE_MAPS_KEY="A.....z"

**Comment by smaeda-ks** [comment, marked as answer]
@zorzpapaduby some characters such as `\` and `"` are escaped upon saving the value, so the original format may be changed. One workaround would be to `base64` encode the value and save that to the env var, then you decode it in code and use that.
